pkg/cmd/grafana-cli/services/api_client.go

```diff
@@ -108,7 +108,7 @@ func (client *GrafanaComClient) DownloadFile(pluginName string, tmpFile *os.File
 		return fmt.Errorf("failed to write to %q: %w", tmpFile.Name(), err)
 	}
 	if len(checksum) > 0 && checksum != fmt.Sprintf("%x", h.Sum(nil)) {
-		return fmt.Errorf("expected SHA256 checksum does not match the downloaded archive - please contact security@grafana.com")
+		return fmt.Errorf("expected SHA256 checksum does not match the downloaded archive (%s: %s) - please contact security@grafana.com", pluginName, url)
```

Comment:
Yeah as @marefr highlighted, this isn't used. No harm in changing it but it serves as a reminder to us to finally clean all this up!

Reply:
have reverted this since it's been cleaned up already!